packages/sdk/codegen.doc.yml

```diff
@@ -6,6 +6,7 @@ hooks:
 config:
   skipComments:
     - "[Internal]"
+    - "[INTERNAL]"
```

Comment:
Should be also skip `[ALPHA]`?

I think we should, it would give us flexibility to do breaking changes in schema without major version bumps in the SDK.

Reply:
That's a good point, I'm adding it.